scrapers/LadyboyGold.yml

```diff
@@ -4,6 +4,12 @@ performerByURL:
     url:
       - ladyboygold.com/index.php
     scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - ladyboygold.com/tour
```

Comment:
``` yaml
     - ladyboygold.com/tour/show_video.php`
```
is probably better to indicate the kind of urls we expect?

Reply:
i wasn't sure if `/tour/show_video.php` was the only URI path or not, so just left it as `/tour` to be more open-ended... I think it's fine as-is for the purpose of `sceneByURL` though?